Dockerfile

```diff
@@ -72,8 +72,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
     imagemagick \
     groff \
     # For PPAs
```

Comment:
I am not sure how useful these existing comments here, but based on that jq should go to common, useful imo